spesifikasjoner/afpant/afpant-model/xsd/dsve.xsd

```diff
@@ -788,4 +789,28 @@
       <xs:element name="kjoeper" type="Aktoer" minOccurs="0" maxOccurs="unbounded"/>
     </xs:sequence>
   </xs:complexType>
+
+  <xs:complexType name="AvbruttSamhandling">
+    <xs:annotation>
+      <xs:documentation>
+        Avbrutt samhandling er en meldings om sendes for tidligere etablert sammhandling hvor forutsettningen for samhandling ikke lenger er tilstede.
+        Typisk kan dette sendes hvor man ikke lenger vil sende intensjonsendring eller kjoepekontraktEndringFraMegler.
+      </xs:documentation>
+    </xs:annotation>
+    <xs:sequence>
+      <xs:element minOccurs="1" maxOccurs="1" name="mottaker" type="Mottaker"/>
+      <xs:element minOccurs="1" maxOccurs="1" name="avsender" type="Avsender"/>
+      <xs:element minOccurs="1" maxOccurs="1" name="begrunnelse" type="xs:string">
```

Comment:
Hvis megler skal sende må vel bankens referanse oppgis for at denne skal kunne matches med bankens lånesak. Skal jo bare av bryte én sak mellom de to partene, ikke alle sakene...

Reply:
<xs:element minOccurs="0" name="referanse" type="xs:string"/>
<xs:element minOccurs="1" name="referanse" type="xs:string"/>
ligger under Mottaker/Avsedner typen? Burde vel holde?

Reply:
Ser at her er mulighet for å legge inn referanse, ja. Men hvis megler skal sende denne meldingen til banken, er det ikke da naturlig at megler oppgir bankens referanse for å gjøre det lettere å matche denne i banksystemet? I så fall er det litt uheldig at referanse er et valgfritt felt i Mottaker-typen. Alternativet her er en egen type med kun de to organisasjonsnumrene og mottakers referanse. Hvis det er meglers referanse som skal sendes er det krav til den i Avsender. Dette kan dere diskutere videre i DSVE-møtene, kanskje.

Reply:
Siden en kontraktforespørsel fra bank kan matche to oppgjør hos megler, må også meglers referanse sendes med slik at banken vet hvilken av de to oppgjørene samhandlingen avbrytes for....